tests/backup/backup_upgrade_test.go

```diff
@@ -1341,11 +1341,11 @@ var _ = Describe("{PXBackupClusterUpgradeTest}", Label(TestCaseLabelsMap[PXBacku
 					err = Inst().V.RefreshDriverEndpoints()
 					log.FailOnError(err, "Refresh Driver Endpoints failed")
 
-					urlToParse := fmt.Sprintf("%s/%s", Inst().StorageDriverUpgradeEndpointURL, Inst().StorageDriverUpgradeEndpointVersion)
-					u, err := url.Parse(urlToParse)
-					log.FailOnError(err, fmt.Sprintf("error parsing PX version the url [%s]", urlToParse))
-					err = Inst().V.ValidateDriver(u.String(), true)
-					dash.VerifyFatal(err, nil, fmt.Sprintf("verify volume driver after upgrade to %s", version))
+					storageNodes := node.GetWorkerNodes()
```

Comment:
GetStorageDriverNodes? All worker nodes need not have PX installed.

Reply:
You got a point. Changing it.